### DISCUSSION TITLE
Search API down?

### DISCUSSION THREAD

**Discussion by ChristopherWalz** [OP-TED/eForms-SDK#825] [Q&A] [answered]
Since the website relaunch (https://ted.europa.eu/en/news/welcome-to-the-new-ted) the API doesn't seem to work any more and is just redirected to the main page: https://ted.europa.eu/api/v3.0/notices/search
Is this going to be fixed?

**Comment by YvesJo** [comment]
Hi,
For this API issue, not related to the SDK, please get in touch directly with the helpdesk ted@publications.europa.eu

**Comment by ChristopherWalz** [comment, marked as answer]
Answer can be found here: https://op.europa.eu/it/web/ted-reusers-workshops/questions-and-answers
